### PR TITLE
Hirad hamza shon tzu/revamping compare accounts table

### PR DESCRIPTION
**Pull request by shontzu-deriv**
## Changes:

Please include a summary of the change and which issue is fixed below:
-   ...

## When you need to add unit test

-   [ ] If this change disrupt current flow
-   [ ] If this change is adding new flow

## When you need to add integration test

-   [ ] If components from external libraries are being used to define the flow, e.g. @deriv/components
-   [ ] If it relies on a very specific set of props with no default behavior for the current component.

## Test coverage checklist (for reviewer)

-   [ ] Ensure utility / function has a test case 
-   [ ] Ensure all the tests are passing

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: packages/cfd/src/Containers/compare-cfds/compare-cfds.tsx [hunk 1]
```diff
@@ -0,0 +1,53 @@
+import React from 'react';
```
Comment (hamza-deriv) [inline packages/cfd/src/Containers/compare-cfds/compare-cfds.tsx]: folder name should `cfd-compare-accounts` or `compare-cfds`?
@hirad-deriv

Comment (hamza-deriv) [reply packages/cfd/src/Containers/compare-cfds/compare-cfds.tsx]: And we have to follow the same convention so all main files like .tsx and .scss one should have name
`cfd-compare-accounts`
Meanwhile the route will only be `compare-cfds`

File: packages/cfd/src/Containers/compare-cfds/compare-cfds.scss [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    justify-content: space-between;
+    &-navigation {
+        display: flex;
+        align-items: center;
+        gap: 8px;
+        cursor: pointer;
+    }
+    &-title {
+        flex: 1;
+        &__mobile {
```
Comment (hamza-deriv) [inline packages/cfd/src/Containers/compare-cfds/compare-cfds.scss]: You can use mixin here and remove this extra class
see if this works
```suggestion
        @include mobile {
```

File: packages/cfd/src/Constants/routes-config.js [hunk 3]
```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import CFD from '../Containers';
 import { routes } from '@deriv/shared';
 import { localize } from '@deriv/translations';
-
+import compareCFDs from '../Containers/compare-cfds';
```
Comment (hamza-deriv) [inline packages/cfd/src/Constants/routes-config.js]: You can use alias here 
```suggestion
import compareCFDs from 'Containers/compare-cfds';
```

File: packages/components/src/components/icon/icons.js [hunk 4]
```diff
@@ -671,6 +671,7 @@ import './rebranding/ic-rebranding-deriv-bot.svg';
 import './rebranding/ic-rebranding-deriv-go-dashboard.svg';
 import './rebranding/ic-rebranding-deriv-trader-dashboard.svg';
 import './rebranding/ic-rebranding-deriv-trader.svg';
+import './rebranding/ic-rebranding-deriv-x.svg';
```
Comment (hirad-deriv) [inline packages/components/src/components/icon/icons.js]: What is the reason of adding this icon ?

**Review by hirad-deriv — APPROVED** [review]

**Pull request merged**